netmaster/master/netmaster.go

```diff
@@ -53,6 +54,7 @@ func SetClusterMode(cm string) error {
 	}
 
 	masterRTCfg.clusterMode = cm
+	SetDNSEnabled(false)
```

Comment:
I had put in this check so that it is explicitly disabled for kubernetes case. The DNS Enabled flag is set after SetClusterMode from execOpts. I'll remove this for consistent handling of setting of options from execOpts. 

/cc @jojimt: You might want to consider updating the flags when running in kubernetes mode